src/WaxSigningApi.ts

```diff
@@ -59,11 +59,16 @@ export class WaxSigningApi {
   public async signing(
     transaction: Transaction,
     serializedTransaction: Uint8Array,
-    noModify = false
+    noModify = false,
+    feeFalback = true
```

Comment:
@danielAlvess we have a typo here that needs to be fixed to line up with the backend: feeFalback->feeFallback